src/workers/parser/getSymbols.js

```diff
@@ -23,40 +23,10 @@ import {
 import { inferClassName } from "./utils/inferClassName";
 import getFunctionName from "./utils/getFunctionName";
 
-let symbolDeclarations = new Map();
-
-export type ClassDeclaration = {|
-  name: string,
-  location: BabelLocation,
-  parent?: ClassDeclaration
-|};
+import type { SimplePath } from "./utils/simple-path";
```

Comment:
just being consistent. i kinda like how it can be more concise, but we dont do it anywhere else